Library/ViewModels/CategorySelectionViewModelTests.swift

```diff
@@ -60,33 +84,293 @@ final class CategorySelectionViewModelTests: TestCase {
       .filmAndVideo
     ])
 
+    let artId = Category.art.intID ?? 0
+    let illustrationId = Category.illustration.intID ?? 0
+    let gamesId = Category.games.intID ?? 0
+    let tabletopId = Category.tabletopGames.intID ?? 0
+    let filmAndVideoId = Category.filmAndVideo.intID ?? 0
+    let documentaryId = Category.documentary.intID ?? 0
+    let unknownId = Category.tabletopGames.intID ?? 0
```

Comment:
Yeah, this is meant to represent a `categoryId` that's not in our hard-coded list of ordered parent categories. I used the `.tableTopGames.intID` because it's not a parent category, so it's not in our list of ordered categories, but it's value can still be converted into an `Int` using `intID`.